library/src/main/java/com/thefuntasty/tasting/ScreenOrientation.kt

```diff
@@ -0,0 +1,5 @@
+package com.thefuntasty.tasting
+
+enum class ScreenOrientation {
+    LEFT, RIGHT, PORTRAIT, SENSOR
```

Comment:
Android používá `landscape`, `reverseLandscape`, `portrait` a `sensor`. Asi bych držel stejné názvosloví.

Reply:
když mrkneš na funkci `changeScreenOrientation` tak tam je vidět jak to děla UI Automator a tam se to jmenuje left, right a natural, to moje mi příjde srozumitelné, ale jestli chceš, tak to změním